src/lib/arch/aarch64/Instruction_decode.cc

```diff
@@ -660,36 +831,42 @@ void Instruction::decode() {
     setInstructionType(InsnType::isPredicate);
   }
   // Uncaught float data assignment for FMOV move to general instructions
-  if (((430 <= metadata_.opcode && metadata_.opcode <= 432) ||
-       (2409 <= metadata_.opcode && metadata_.opcode <= 2429)) &&
+  if (((Opcode::AArch64_FMOVD0 <= metadata_.opcode &&
+        metadata_.opcode <= Opcode::AArch64_FMOVS0) ||
+       (Opcode::AArch64_FMOVDXHighr <= metadata_.opcode &&
+        metadata_.opcode <= Opcode::AArch64_FMOVv8f16_ns)) &&
```

Comment:
Aren't the `_ns` instructions moving immediates into vector registers? (and thus aren't "move to general instructions")

Reply:
For this I replaced the numbers with Opcodes (and referred to the previous Capstone version to make sure they aligned). If this is the case then we can remove the `_ns` ones from this catch

Reply:
Double-checked and we should remove `_ns` from this catch

Reply:
Updated catch